Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -182,6 +209,17 @@
   categories:
   - UplinkWearables
 
+  - type: listing
+  id: UplinkClothingDuffelbagSyndicateYukon
```

YAMLlint (1.35.1)

[error] duplication of key "id" in mapping

(key-duplicates)